Cargo.toml

```diff
@@ -85,3 +85,6 @@ parquet-format-async-temp = { git = "https://github.com/MaterializeInc/parquet-f
 # Waiting on https://github.com/tokio-rs/prost/pull/576.
 prost = { git = "https://github.com/MaterializeInc/prost.git" }
 prost-types = { git = "https://github.com/MaterializeInc/prost.git" }
+
+[patch."https://github.com/MaterializeInc/rust-postgres"]
+postgres-types = { path = "/Users/martin/work/rust-postgres/postgres-types" }
```

Comment:
@mkysel Can I get the contents of this patch?

Reply:
``` $ git diff
diff --git a/postgres-types/src/type_gen.rs b/postgres-types/src/type_gen.rs
index 7a16e8d..b09ee35 100644
--- a/postgres-types/src/type_gen.rs
+++ b/postgres-types/src/type_gen.rs
@@ -557,7 +557,7 @@ impl Inner {
                 &Kind::Simple
             }
             Inner::Int2Vector => {
-                &Kind::Array(Type(Inner::Int2))
+                &Kind::Simple
             }
             Inner::Int4 => {
                 &Kind::Simple